src/fiskaltrust.ifPOS/v1/de/Models/TseInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace fiskaltrust.ifPOS.v1.de
+{
+    [DataContract]
+    public class TseInfo
+    {
+        [DataMember(Order = 1)]
+        public long MaxNumberOfClients { get; set; }
+
+        [DataMember(Order = 2)]
+        public long CurrentNumberOfClients { get; set; }
+
+        [DataMember(Order = 3)]
+        public long MaxNumberOfTransactions { get; set; }
+
+        [DataMember(Order = 4)]
+        public long CurrentNumberOfTransactions { get; set; }
+
+        [DataMember(Order = 5)]
+        public IEnumerable<CertificateInfo> Certificates { get; set; }
+
+        [DataMember(Order = 6)]
+        public IEnumerable<string> SerialNumbersBase64 { get; set; }
+
+        [DataMember(Order = 7)]
+        public TseState CurrentState { get; set; }
+
+        /// <summary>
```

Comment:
Multiline summaries are formatted weirdly by VS, could we add a one-line summary and set this as the `<description>`?